src/project.ts

```diff
@@ -95,7 +95,7 @@ export async function getTree(
   if (path === "/") {
     path = "";
   }
-  return api.get(`projects/${urn}/tree/${commit}/${path}`, {}, config);
+  return api.get(`projects/${urn}/tree/${config.seed.link.id}/${commit}/${path}`, {}, config);
```

Comment:
I don't think this is going to do it, as this is trying to get the remote identified by the seed id, which most likely has no remotes of its own. By default, we should be using the first maintainer/delegate's peer-id which I think was the previous behavior of the http-api, and is what is returned as `remote` in the project_info endpoint.